portal/server/routes/contract/about/industryFields/index.js

```diff
@@ -7,24 +7,36 @@ const industryFields = (postedSubmissionDetails, industrySectors) => {
 
   let industrySectorObj;
 
-  if (industrySectorCode) {
+  if (!industrySectorCode) {
+    submissionDetails['industry-sector'] = {};
+  } else {
     industrySectorObj = getIndustrySectorById(industrySectors, industrySectorCode);
 
-    submissionDetails['industry-sector'] = {
-      code: industrySectorCode,
-      name: getIndustrySectorById(industrySectors, industrySectorCode).name,
-    };
-  } else {
-    submissionDetails['industry-sector'] = {};
+    if (industrySectorObj) {
+      submissionDetails['industry-sector'] = {
+        code: industrySectorCode,
+        name: industrySectorObj.name,
+      };
+    } else {
```

Comment:
could we do this without the else? 

`submissionDetails['industry-sector'] = {};

 if (industrySectorObj) {
      submissionDetails['industry-sector'] = {
        code: industrySectorCode,
        name: industrySectorObj.name,
      };
    }
`

Reply:
Unfortunately not - left a comment in the code on line 21, in the else.

If no industry sector object is found via `getIndustrySectorById`, `submissionDetails['industry-sector']` will be set to undefined when it should be an empty object.